apps/system-e2e/src/integration/air-discount-scheme/homepage.spec.ts

```diff
@@ -0,0 +1,58 @@
+// import { idsLogin } from '../../support/commands'
+import {
+  aliasQuery,
+  getFakeUser,
+  getFamily,
+  getBaseUrl,
+  getDiscountData,
+} from '../../support/utils'
+
+describe('Home page', () => {
+  const baseUrl = getBaseUrl(Cypress.config())
+
+  const fakeUser = getFakeUser(Cypress.env('fakeUsers'), 'gervimaður afríka')
+  const family = getFamily(fakeUser)
+
+  beforeEach(() => {
+    cy.idsLogin({
+      phoneNumber: fakeUser.phoneNumber,
+      urlPath: '/min-rettindi',
+    })
+    cy.intercept('POST', `${baseUrl}/api/graphql`, (req) => {
+      aliasQuery(req, 'DiscountsQuery')
+    })
+  })
+
+  it(`should return discounts response array`, () => {
+    cy.visit('/min-rettindi')
+    cy.wait('@DiscountsQuery', { timeout: 20000 }).then((data) => {
+      assert.isNotNull(
+        data.response?.body.data.discounts,
+        'Discounts response exists',
+      )
+    })
+  })
+
+  it(`should have user ${fakeUser.name} with valid discount code`, () => {
+    cy.visit('/min-rettindi')
+    cy.wait('@DiscountsQuery', { timeout: 20000 }).then((data) => {
+      const { discountUser, discounts } = getDiscountData(
+        fakeUser,
+        data.response,
+      )
+      expect(
+        discounts.map((e) => e.user.name),
```

Comment:
Why are concerned with the children?

Reply:
the Discounts gql response includes an array of  parents and their children (if any). This particular gervimaður has a child so I also added a test to validate that the child exists in the discounts array.

Reply:
I think the presence of children is not relevant to the discount applicability and only has the potential to break for no good reason if we change the user. Remove this expectation altogether?

Reply:
yeah, removed